aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -526,6 +532,48 @@ Resources:
 <%  unless load_balancer -%>
           - !ImportValue VPC-ELBSecurityGroup # Accept HTTP traffic on daemon
 <%  end -%>
+  DaemonStorageUtilizationAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      AlarmActions:
+        - <%= rack_env?(:production) ? 'arn:aws:sns:us-east-1:475661607190:CDO-Urgent' : 'arn:aws:sns:us-east-1:475661607190:CDO-LowPriority' %>
+      AlarmDescription: Send page if daemon storage utilization exceeds 80% for an hour
+      AlarmName: <%="#{rack_env}_daemon_low_disk_space" %>
```

Comment:
We can't entirely assume there will only be a single stack in each environment (e.g., `adhoc` environment has multiple stacks), and [`AlarmName`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cw-alarm.html#cfn-cloudwatch-alarms-alarmname) is a [Name Type](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-name.html) so it must be unique across all stacks, so it's probably better to set `AlarmName` based on the StackName, e.g.:

```
      AlarmName: !Sub "${AWS::StackName}__daemon_low_disk_space"
```